### ISSUE TITLE
Not working for Google employees.

### ISSUE THREAD

**Issue by NeilFraser** [GMaiolo/remove-w3schools#6]
w3schools results are showing up in my search results.  A quick check doesn't see any problems:

* No messages in the console.
* document.querySelectorAll('div.g:not([style*="display:none"]):not([style*="display: none"]) .r > a[href*="www.w3schools.com"]') correctly returns: NodeList(3) [a, a, a]
* document.getElementById('main') correctly returns: <div id=​"main">​…​</div>​

Not debugging further until there's confirmation that it's not just me.
Chrome Version 64.0.3282.167 (Official Build) (64-bit) OS X.

**Comment by GMaiolo** [comment]
Can you provide the search link?

**Comment by NeilFraser** [comment]
https://www.google.com/search?q=frameset

**Comment by GMaiolo** [comment]
I'm having it successfully removed.
Was thinking that the problem was somewhat related to #2 but your link seems to be common.

**Comment by NeilFraser** [comment]
More debugging:

Removed W3Schools 3.0.12, readded it, no change.
Added both other W3School removal extensions, none of them work.

Best guess at this point is that these extensions are being silently blocked from executing by my employer (Google).  After all, they are a security risk in that they can edit any page on google.com, which includes, well, everything here.  But I don't have evidence of this at the moment.

**Comment by NeilFraser** [comment]
Yes, it is definitely on my end.  I wrote a trivial Tampermonkey script that console logs 'hello' when on my personal website.  Works.  Then changed the triggering URL to google.com.  Stopped working.

Google does not appear to allow extensions on employee computers to mess with Google.  I understand the security reasons.  The extension authors would have read/write access to our intranet.  Bummer.

Thanks for your help.  Closing.
